File: test/tests/code/io_stuff.java
import java.io.InputStreamReader;
import java.io.BufferedReader;
import java.io.IOException;

class IOStuff
{
	public static void ioStuff()
	{
		System.out.println("This is stdout data.");
		System.err.println("This is stderr data.");
		
		try
		{
			BufferedReader stdin = new BufferedReader(new InputStreamReader(System.in));
			String data = stdin.readLine();
			System.out.println("The stdin data: " + data);
		}
		catch (IOException e) {
			System.err.println("FAILED TO READ STDIN DATA!");
		}
	}
}

File: test/tests/code/io_stuff.cpp
#include <iostream>
#include <string>

void ioStuff()
{
	std::cout << "This is stdout data." << std::endl;
	std::clog << "This is stderr data." << std::endl;
	
	std::string data = "";
	std::getline(std::cin, data);
	std::cout << "The stdin data: " << data << std::endl;
}
